shims/common/src/main/scala/io/glutenproject/GlutenConfig.scala

```diff
@@ -365,7 +367,12 @@ object GlutenConfig {
       throw new UnsupportedOperationException(s"${GlutenConfig.GLUTEN_OFFHEAP_SIZE_KEY} is not set")
     }
     val offHeapSize = conf.getSizeAsBytes(GlutenConfig.GLUTEN_OFFHEAP_SIZE_KEY)
-    generatedMap.put(GlutenConfig.GLUTEN_OFFHEAP_SIZE_IN_BYTES_KEY, offHeapSize.toString)
+    val glutenOffheapPercent = conf.getDouble(
+      GlutenConfig.GLUTEN_OFFHEAP_PERCENT_KEY,
+      GlutenConfig.GLUTEN_DEFAULT_OFFHEAP_PERCENT)
+    generatedMap.put(
+      GlutenConfig.GLUTEN_OFFHEAP_SIZE_IN_BYTES_KEY,
```

Comment:
I recommend to use a new key like
`spark.gluten.sql.columnar.backend.velox.memoryCap`.

`spark.gluten.memory.offHeap.size.in.bytes` is supposed to be just `spark.gluten.memory.offHeap.size` in bytes.

Reply:
okey, `spark.gluten.sql.columnar.backend.velox.memoryCap` is better.